node.gyp

```diff
@@ -616,13 +608,9 @@
       'target_name': 'cctest',
       'type': 'executable',
       'dependencies': [ 'deps/gtest/gtest.gyp:gtest' ],
-      'conditions': [
-        [ 'node_shared_v8=="false"', {
-          'dependencies': [ 'deps/v8/tools/gyp/v8.gyp:v8' ],
```

Comment:
You should add the v8.gyp:v8 dependency to the dependencies list three lines up.  I suspect that is what is breaking the Windows build right now.